File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    sound = ""
    if number % 3 == 0:
        sound += "Pling"
    if number % 5 == 0:
        sound += "Plang"
    if number % 7 == 0:
        sound += "Plong"
    if number % 3 and number % 5 and number % 7:
        return str(number)
    else:
        return sound

File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"
#include<string>
#include<iostream>
using namespace std;

namespace raindrops {
    string convert(const int &num){
        string s = "";
        if(num % 3 == 0)
            s.insert(s.size(), "Pling");
        if(num % 5 == 0)
            s.insert(s.size(), "Plang");
        if(num % 7 == 0)
            s.insert(s.size(), "Plong");
       
        
        if(s.empty()){
             string Snum = to_string(num);
            return Snum;
        }
        return s;
    }    

}  // namespace raindrops
